examples/server/server.cpp

```diff
@@ -649,7 +649,7 @@ int main(int argc, char **argv)
                       {"tokens_predicted", llama.num_tokens_predicted}};
                   return res.set_content(data.dump(), "application/json");
                 }
-                catch (json::exception e)
+                catch (json::exception const &e)
```

Comment:
i think we have a `const` on the right side consensus.

Reply:
Could you please show me an example of what you mean? `json::exception & const e` doesn't work nor does `json::exception & e const`

Reply:
You made me think I was crazy for a second there!